website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -108,21 +108,43 @@ models:
 
 ## Materialized views
 
-The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html).
-Indexes are the only configuration that is specific to `dbt-postgres`.
-The remaining configuration follows the general [materialized view](/docs/build/materializations#materialized-view) configuration.
-There are also some limitations that we hope to address in the next version.
+The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html)
+with the following configuration parameters:
 
-### Monitored configuration changes
+| Parameter                 | Type       | Required | Default   | Change Monitoring Support | Reference           |
+|---------------------------|------------|----------|-----------|---------------------------|---------------------|
+| `on_configuration_change` | STRING     | NO       | `'apply'` | N/A                       |                     |
```

Comment:
Similarly to the link to "Indexes", do you want to link to the docs on `on_configuration_change ` that are featured [here](https://docs.getdbt.com/docs/build/materializations#materialized-view)?

Reply:
Honestly, I want to break that out as its own section, and then link to that both here and in materialized views. It probably makes sense to add the link here for now though.